explainability-core/src/main/java/org/kie/trustyai/metrics/drift/fouriermmd/DriftResult.java

```diff
@@ -0,0 +1,30 @@
+package org.kie.trustyai.metrics.drift.fouriermmd;
+
+/*
+ * Return value from FourierMMD.execute()
+ */
+
+public class DriftResult {
```

Comment:
Please name this more specifically to tie it to the FourierMMD algo

Reply:
renamed to FourierMMDResult